src/input.ml

```diff
@@ -11,7 +11,7 @@ let set_input filename =
 
 (** Get the name of the module from the name of the file. *)
 let get_module_name () =
-  Filename.chop_extension (Filename.basename !input_file)
+  Output.low_dash (Filename.chop_extension (Filename.basename !input_file))
```

Comment:
What is this function "low_dash"?

Reply:
The goal is to avoid "Invalid name directive ignored" warning : usual OpenTheory filenames have "-" characters, which were transformed while writing to Dedukti.

Instead of changing the filenames similarly, I chose to transform "-" to low dashes to maintain readability.